test/Microsoft.Health.SqlServer.Web/Properties/launchSettings.json

```diff
@@ -4,9 +4,8 @@
       "commandName": "Project",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "development",
-        "SqlServer:Initialize": "false",
         "SqlServer:SchemaOptions:AutomaticUpdatesEnabled": "true",
-        "SqlServer:AllowDatabaseCreation": "false",
+        "SqlServer:AllowDatabaseCreation": "true",
```

Comment:
We might have to keep this to 'false' to not allow the E2E tests to try creating database until we add the ability to create the database in Github actions. I think E2E tests are currently failing on this PR due to this reason.

Reply:
Ah, thank you for looking into this. I originally set this to `true` to get the unit tests to pass. I'll figure out a different way around the problem!

Reply:
Reverted this to `false` in 253f2af.